src/faq.md

```diff
@@ -446,6 +446,16 @@ The reference runner and several other CWL implementations support running
 those Docker format containers using the Singularity engine. Directly
 specifying a Singularity format container is not part of the CWL standards.
 
+## Shell Command Requirement
+`ShellCommandRequirement` in CWL is used to convert a list of arguments into a string containing a shell command line.  
+Each item in the argument list must be joined into a string separated by single spaces and quoted to prevent intepretation by the shell, unless `CommandLineBinding` for that argument contains `shellQuote: false`.  
```

Comment:
If you're going to copy text from the specification, you have to at least reword it so it is speaking to the right audience.

The specification is speaking to people who are writing code to run CWL.  So when it says "it _must_ do something" it is saying their CWL runner must do this.

For the audience of CWL users, you want to say what _will_ happen.  So this needs to be change to say "...each item _will_ be joined..."